AK/URL.cpp

```diff
@@ -260,16 +260,25 @@ bool URL::is_special_scheme(StringView scheme)
     return scheme.is_one_of("ftp", "file", "http", "https", "ws", "wss");
 }
 
-DeprecatedString URL::serialize_path() const
+// https://url.spec.whatwg.org/#url-path-serializer
+DeprecatedString URL::serialize_path(ApplyPercentDecoding apply_percent_decoding) const
 {
+    // If url has an opaque path, then return url’s path.
+    // FIXME: Reimplement this step once we modernize the URL implementation to meet the spec.
     if (cannot_be_a_base_url())
         return m_paths[0];
-    StringBuilder builder;
-    for (auto& path : m_paths) {
-        builder.append('/');
-        builder.append(percent_decode(path));
+
+    // Let output be the empty string.
```

Comment:
Hm. Kind of weird that the spec has a bulleted list for this instead of a numbered list. Might be worth a spec issue in https://github.com/whatwg/html  if you're feeling speccy :)

Reply:
Yea I wasn't too sure if this was intentional too. I'll file an issue there 👍

Reply:
Opened an issue at https://github.com/whatwg/url/issues/786